.github/workflows/tests-cleanup.yml

```
@@ -0,0 +1,77 @@
+name: TestsCleanUp
+on:
+  push:
+    branches:      
+      - tf-*      
+  schedule:
+    - cron: '0 0 * * *' # Runs dayly at 12:00:00
+  workflow_dispatch:
+jobs:
+  build:    
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1    
+    - name: Run pws script to delete SQS transport acceptance tests AWS artifacts
+      env: 
+        LIMITED_DELETE_RIGHTS_ACCOUNT_KEY: ${{ secrets.TMP_CHRISTIAN_AWS_ACCESS_KEY_ID }}
+        LIMITED_DELETE_RIGHTS_ACCOUNT_SECRET_KEY: ${{ secrets.TMP_CHRISTIAN_AWS_SECRET_KEY_ID }}
```

Before merging, not immediately - these are the correct secret names:
```suggestion
        LIMITED_DELETE_RIGHTS_ACCOUNT_KEY: ${{ secrets.CLEANUP_AWS_ACCESS_KEY_ID }}
        LIMITED_DELETE_RIGHTS_ACCOUNT_SECRET_KEY: ${{ secrets.CLEANUP_AWS_SECRET_ACCESS_KEY }}
```

(And delete the temp secrets as well)